geode-core/src/distributedTest/java/org/apache/geode/management/internal/cli/NetstatDUnitTest.java

```diff
@@ -161,6 +161,8 @@ public void testOutputToFileForOneGroup() throws Exception {
     assertThat(lines.get(4).trim().split("[,\\s]+")).containsExactly("server-1");
   }
 
+  // This test runs OK on JDK8 but takes a very long time on JDK8
```

Comment:
Just missing JDK11 instead of JDK8 twice

Reply:
Oops, typo in the comment.